lib/Containers/MainContainer.js

```diff
@@ -1,16 +1,32 @@
 import React, { PureComponent } from 'react';
-import { Header } from '@bpanel/bpanel-ui';
+import { Header, createNestedViews } from '@bpanel/bpanel-ui';
+
+import CreateNew from './CreateNew';
+import SendReceive from './SendReceive';
+import Overview from './Overview';
 
 export default class MainContainer extends PureComponent {
   constructor(props) {
     super(props);
   }
 
   render() {
+    const { match } = this.props;
+
+    // TODO: figure out how to get subitems on sidebar to work
```

Comment:
Hopefully the docs help with this: https://bpanel.org/docs/ui-utilities.html#createnestedviews
If there's anything that's not clear, overlooked, or could be clarified, let me know! (or submit as PR to bpanel-docs 😁). There's also a sample implementation in the @bpanel/bui plugin.

Reply:
It looks like you're basically there though!